docs/stream-api.md

```diff
@@ -524,6 +530,24 @@ $bunnyHttpClient->request(
 
     This endpoint will return a `400` status code if all available resolutions for the video are passed to `resolutionsToDelete`, as there must be at least one resolution available after cleanup.
 
+#### [Trigger Smart Actions](https://docs.bunny.net/reference/video_smartgenerate)
+
+```php
+$bunnyHttpClient->request(
+    new \ToshY\BunnyNet\Model\Api\Stream\ManageVideos\TriggerSmartActions(
+        libraryId: 1,
+        videoId: 'e7e9b99a-ea2a-434a-b200-f6615e7b6abd',
+        body: [
+            'generateTitle' => true,
+            'generateDescription' => true,
+            'generateChapters' => true,
+            'generateMoments' => true,
+            'sourceLanguage' => 'en',
+        ],
+    )
+);
+```
+
```

Comment:
Add note that when transcribing is not used, or no captions are available, this will return `400` error.